data/tr3/ship/cfg/catalog_lara_states.csv

```diff
@@ -0,0 +1 @@
+
```

Comment:
Just checking this is intended to be empty, while anims isn't. Not important for the time being of course.